src/TypeDefinition.js

```diff
@@ -185,6 +185,11 @@ export type NavigationScreenOptions = {
    * Options passed to the drawer for this screen.
    */
   drawer?: NavigationScreenOption<DrawerConfig>;
+  /**
+   * Whether you can use gestures to dismiss this screen.
+   * Defaults to true on iOS, false on Android.
+   */
+  gesturesEnabled?: bool,
```

Comment:
can you put it inside another object `cardStack`? we'll likely add more options in future and it'll be useful to access navigation state and params.